build.gradle.kts

```diff
@@ -281,8 +261,27 @@ subprojects {
                     applicationIdSuffix = ".debug"
                 }
 
-                maybeCreate("debugProd").apply {
+                create("debugProd") {
                     applicationIdSuffix = ".debug"
+                    isDebuggable = true
+                    isPseudoLocalesEnabled = true
+                    enableUnitTestCoverage = false
+                    enableAndroidTestCoverage = false
+                    ext.set("alwaysUpdateBuildId", false)
+
+                    buildConfigField("String", "SERVER_MAIN_URL", SERVER_MAIN_URL_PROD)
+                    buildConfigField("String", "SERVER_API_URL", SERVER_API_URL_PROD)
+                    buildConfigField("String", "SERVER_CACHE_URL", SERVER_CACHE_URL_PROD)
+                    buildConfigField("String", "SERVER_STATIC_URL", SERVER_STATIC_URL_PROD)
+                    buildConfigField("String", "SERVER_SHARING_URL", SERVER_SHARING_URL_PROD)
+                    buildConfigField("String", "SERVER_SHORT_URL", SERVER_SHORT_URL_PROD)
+                    buildConfigField("String", "WEB_BASE_HOST", WEB_BASE_HOST_PROD)
+                    buildConfigField("String", "SERVER_LIST_URL", SERVER_LIST_URL_PROD)
+                    buildConfigField("String", "SERVER_LIST_HOST", SERVER_LIST_HOST_PROD)
+
+                    signingConfig = signingConfigs.getByName("debug")
+
+                    matchingFallbacks += listOf("debug")
```

Comment:
I could be doing something wrong but `debugProd` seems to be pointing to staging and not production. 

<img width="1530" alt="Screenshot 2024-08-19 at 9 44 22 PM" src="https://github.com/user-attachments/assets/8899c40c-efb0-4b2c-b75b-f5ce54c37925">

Reply:
Ohhh… I know what's going on. We don't use DI for build config so these properties are redefined for every module. I'll work on a fix. It should also improve build times because we won't be generating build config for every library module.